src/index.ts

```diff
@@ -91,7 +90,7 @@ app.use(
     store: redisStore,
     cookie: {
       secure: isDev ? false : true,
-      sameSite: isProd ? 'strict' : 'none',
+      sameSite: 'strict',
```

Comment:
This will break SIWE on preview app.
```suggestion
      sameSite: isProd ? 'strict' : 'none',
```

Reply:
'strict' seems like it is needed for local dev, I guess we need to work with it manually on local dev.